instance/models/openedx_appserver.py

```diff
@@ -144,6 +154,8 @@ def set_field_defaults(self):
         """
         Set default values.
         """
+        if not self.github_admin_organizations and settings.DEFAULT_ADMIN_ORGANIZATION:
```

Comment:
Yes, when I said "the way it is now", I meant to leave `if not self.github_admin_organizations and settings.DEFAULT_ADMIN_ORGANIZATION:` as @smarnach wrote (as the diff shows now). Sorry for the ambiguity.